src/utils/style-formatter.js

```diff
@@ -79,4 +78,18 @@ export default {
 
     return styles;
   },
+  setFontSize(button, style) {
```

Comment:
Could destructureto label and fontSize

Reply:
Yes everything needs to be cleaned up still